### PR TITLE
Speed up repo manager end to end test

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #901

### CODE REVIEW

File: ga4gh/cli.py [hunk 1]
```diff
@@ -1400,9 +1400,9 @@ def getRepoParser():
     return parser
 
 
-def repo_main():
+def repo_main(args=None):
     parser = getRepoParser()
-    args = parser.parse_args()
+    args = parser.parse_args(args)
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: It's confusing to reuse a function parameter as a local variable; can you rename the second 'args' to something please? (parsedArgs, eg.)

**Comment by jeromekelleher** [comment]
Nice, +1.

**Comment by dcolligan** [comment]
@jeromekelleher updated

**Comment by jeromekelleher** [comment]
Merging.

**Pull request merged**